puzzle/plugins/gemini/variant_mixin.py

```diff
@@ -2,19 +2,45 @@
 
 from gemini import GeminiQuery
 
+from puzzle.plugins import BaseVariantMixin
+
 from puzzle.models import (Compound, Variant, Gene, Genotype, Transcript,)
 
 from puzzle.utils import (get_most_severe_consequence, get_omim_number,
-                          get_cytoband_coord, get_gene_info)
+                          get_cytoband_coord)
 
 
 logger = logging.getLogger(__name__)
 
-class VariantMixin(object):
+class VariantMixin(BaseVariantMixin):
     """Class to store variant specific functions for gemini plugin"""
 
+
+    def build_gemini_query(self, query, extra_info):
+        """Append sql to a gemini query
+        
+        Args:
+            query(str): The gemini query
+            extra_info(str): The text that should be added
+        
+        Return:
+            extended_query(str)
+        """
+        if 'WHERE' in query:
+            return "{0} AND {1}".format(query, extra_info)
+        else:
+            return "{0} WHERE {1}".format(query, extra_info)
```

Comment:
The info comes from the variants function where the filters are checked.
This is just a small help function to avoid code repetition.
There is no syntax checking since the input strings ar all hard coded...